BookingProject/BookingWebAPI/BookingWebApi.Application/DTOs/AppUserDTOs/LoginDto.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BookingWebApi.Application.DTOs.AppUserDTOs
+{
+    public class LoginDto
+    {
+        public string? Email { get; set; }
+        public string? Password { get; set; }
```

Comment:
When you use records remove setters, because DTOs should not have setters, they are immutable by design. It should be required and `init` only